src/components/Map.vue

```diff
@@ -23,15 +32,12 @@ export default {
     LGeoJson,
     LMap,
     LRectangle,
-    LTileLayer
+    'l-wms-tile-layer': LWMSTileLayer
```

Comment:
```suggestion
    LWMSTileLayer
```

Reply:
See the new commit. I was not able to get this change to compile.

Reply:
Ah, yes, I forgot... it should work if you also use `LWMSTileLayer` instead of `l-wms-tile-layer` in line 5. Or alternatively, use `l-w-m-s-tile-layer`. This is due to Vue converting from CamelCase to kebab-case stupidly...